src/slider/index.css

```diff
@@ -0,0 +1,35 @@
+gb-slider.gb-ui,
+[data-is="gb-slider"].gb-ui {
+  max-width: 300px;
+  height: 18px;
+  margin: 20px auto 30px;
+  display: block;
+}
+
+gb-slider.gb-ui .gb-slider__base,
+[data-is="gb-slider"].gb-ui .gb-slider__base {
+  width: 100%;
+  height: 100%;
+  position: relative;
+  border-radius: 25px;
+  background: blue;
+}
+
+gb-slider.gb-ui .gb-slider__handle,
+[data-is="gb-slider"].gb-ui .gb-slider__handle {
+  position: absolute;
+  height: 28px;
```

Comment:
again, can we adapt to the heights and widths set by the customer instead of setting them ourselves?